networking/kvm.go

```diff
@@ -120,6 +125,15 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf, interfaceNumber
 	if err := netlink.LinkAdd(link); err != nil {
 		return nil, errwrap.Wrap(errors.New("cannot create macvtap interface"), err)
 	}
+
+	// TODO: duplicate following lines for ipv6 support, when it will be added in other places
+	ipv4SysctlValueName := fmt.Sprintf(IPv4InterfaceArpProxySysctlTemplate, interfaceName)
+	if _, err := cnisysctl.Sysctl(ipv4SysctlValueName, "1"); err != nil {
```

Comment:
Good question. I think that this is sane for each of macvlan/macvtap modes, but changing this for optional (with enabled as default) option seems legit as a follow up.

Would you propose such update on `cni` side? I would mirror this there then.